1224-user_administration/xxxx-user_is_assigned_creator_role_according_to_settings_for_their_customer_institution.feature

```diff
@@ -0,0 +1,10 @@
+    @xxxx
+    Scenario Outline: User is assigned the Creator role according to settings for their Customer Institution
+        Given that the User is logged in
+        And the user's Customer Institution is set to assign the Creator role for users with Feide roles <FeideRole>
+        And the user's Feide account has roles <FeideRole>
+        Then the user is assigned the Creator role
+        Examples:
+            | FeideRole                 |
+            | member, employee, faculty |
+            | member, employee, staff   |
```

Comment:
Hvis vi tenker på scenarioene her som at de skal kunne testes i Cypress, så føler jeg at dette scenarioet ikke hører hjemme

Reply:
det er jo en funksjonalitet som trengs, så den bør jo representeres av et scenario av noe slag?
i testene kan vi jo styre selv hvilke settings som gjelder for hver institusjon, så dette kan jo testes ved å se at brukeren får opp valget om å opprette publikasjon når tilhørende institusjon har en verdi er satt, og ikke når den ikke er satt.
Bruken av `FeideRole` trengs for at den på backend som tar oppgaven skjønner hvordan dette skal virke opp mot Feide-bruker

Reply:
ja, jeg er jo enig i at det er info som bør bli formidlet. @Eiriknil får finne ut om denne er testbar når han kommer så langt da 😛

Reply:
Jeg har fått en forklaring av Jan på hva som skjer ved første gangs pålogging og det ser ut som det skal være mulig å teste dette